test/test-coordinator.js

```diff
@@ -245,6 +247,19 @@ TestCoordinator.prototype.callEndpoint = function callEndpoint(endpoint, body, c
 TestCoordinator.prototype.getMembership = function getMembership() {
     return this.getFakeNodes().map(function(node) {
         return node.toMemberInfo();
+    }).concat(this.extraMembers);
+};
+
+TestCoordinator.prototype.extraMember = function(address, status, incarnationNumber) {
```

Comment:
Why not addMember? is that already used?
I think that the word extra doesn't convey the adding of the member